docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/StorageManagerPathPropExample.tsx

```diff
@@ -0,0 +1,13 @@
+import { StorageManager } from '@aws-amplify/ui-react-storage';
+
+export const StorageManagerPathPropExample = () => {
+  return (
+    <StorageManager
+      accessLevel="guest"
+      // uploaded files can be found in S3 inside `guest/images/`
```

Comment:
Not exactly correct, it would be `public/images/` because of course

Reply:
Pro tip: `suggestion` 😏

Reply:
```suggestion
      // uploaded files can be found in S3 inside `public/images/`
```